plan_test.go

```diff
@@ -0,0 +1,40 @@
+package stripe
+
+import (
+	"strconv"
+	"testing"
+
+	assert "github.com/stretchr/testify/require"
+)
+
+func TestPlanParamsAppendTo(t *testing.T) {
```

Comment:
Note that a major change to testing strategy here is that parameter serialization starts getting its own first class tests. This represents a huge share of stripe-go by line count and is very prone to error (we regularly get user patches for this stuff), and yet has very little in the way of direct testing.

This test serializes parameters to a form body, and then immediately checks the result.